src/main/kotlin/com/rooplor/classcraftbackend/controllers/ClassController.kt

```diff
@@ -98,4 +98,10 @@ class ClassController
         fun removeClass(
             @PathVariable id: String,
         ): Unit = service.deleteClass(id)
+
+        @Operation(summary = "Find classes by owners")
+        @GetMapping("/owners")
+        fun findClassByOwners(
+            @RequestParam owners: List<String>,
+        ): List<ClassListDTO> = listMapper.mapList(service.findClassByOwners(owners), ClassListDTO::class.java, modelMapper)
```

Comment:
what about create class? It should modify to add user when create?

Reply:
I updated classDTO already for pass user who created this class